src/Pages/Detail/Detail.js

```diff
@@ -32,17 +32,17 @@ class Detail extends Component {
   }
 
   componentDidMount() {
-    fetch(`http://10.58.2.217:8000/products/${this.props.match.params.id}`)
+    fetch(`http://3.17.134.84:8000/products/${this.props.match.params.id}`)
```

Comment:
```suggestion
    fetch(`${API}/products/${this.props.match.params.id}`)
```
src 폴더에 config.js 파일 만들고 API 주소 등 별도로 관리해주세요

Reply:
헛 시도해보겠슴다!